tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -74,8 +74,6 @@ def parameters(cls):
     def configure(self):
         box_outputs = ['classes_out', 'scores_out', 'boxes_out']
         detection_out = 'detection_out'
-        if contains_all(self.launcher_config, [*box_outputs, detection_out]):
-            raise ConfigError('only detection output or [{}] should be provided'.format(', '.join(box_outputs)))
```

Comment:
but we still need to check that both output formats are no provided toghther
What if `detection_out` and classes_out', 'scores_out', 'boxes_out' are not empty at the same time?

Reply:
Fixed, please review again